tests/test_lib_applications.py

```diff
@@ -77,8 +80,10 @@ class LUKSApplicationTestCase(MyTestCase):
     def test_01_get_options(self):
         # Can run as class
         options = LUKSApplication.get_options()
-        self.assertEqual(options["required"], [])
-        self.assertEqual(options["optional"], ['slot', 'partition'])
+        self.assertIn("slot", options)
+        self.assertIn("partition", options)
+        self.assertEqual("int", options.get("slot").get("type"))
+        self.assertEqual("str", options.get("partition").get("type"))
```

Comment:
Should we mention the change in the returned values in the READ_BEFORE_UPDATE file?

Reply:
I think the change is only in the displaying of the UI. The value is still stored in the database as a string. So reading old valus does not break anything as far as I remember.

Reply:
Will not do.

Reply:
I meant, that the result of the API-call changed. From:
```
...
  "options": {
    "required": [],
    "optional": ["slot", "partition"]
  }
...
```
to:
```
...
  "options": {
    "slot": {
      "type": "int"
    }
    "partition": {
      "type": "str"
    }
  }
...
```
It is only for the UI and probably no-one ever used this but it is a change in the API...
We should think about how we communicate changes in the API.

Reply:
OK, I see. Will do.

Reply:
done in 003a89ea
Will merge when tests are done.